dive_core/capture_data.cpp

```diff
@@ -1143,7 +1143,8 @@ CaptureData::LoadResult CaptureData::LoadAdrenoRdFile(FileReader &capture_file)
         case RD_CHIP_ID:
         {
             // If it wasn't set already by a RD_GPU_ID
-            if (GetGPUID() == 0)
+            // Or if it was an invalid gpu_id, which leads to a kGPUVariantNone
```

Comment:
this is in another pr already?

Reply:
Yes I realized as I created this PR that I accidentally included the commit from the other PR. By the time I managed to fix it (2 minutes later!), you already made this comment. You're too fast!